samples/boards/nrf/clock_control/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: Clock control sample for nRF
+tests:
```

Comment:
Please add harness console and expected regex to enable automatic verification of this sample, e.g:
https://github.com/nrfconnect/sdk-zephyr/blob/bc15a7e16a7266f66a229d1673177e45f0457652/samples/drivers/adc/adc_dt/sample.yaml#L38-L44